src/index.tsx

```diff
@@ -3,11 +3,20 @@ import React from "react";
 import ReactDOM from "react-dom";
 import "./index.css";
 import App from "./App";
+import { QueryParamProvider } from "use-query-params";
+import { BrowserRouter, Route } from "react-router-dom";
 
 loadDevTools(() =>
   ReactDOM.render(
     <React.StrictMode>
-      <App />,
+      <BrowserRouter>
+        <QueryParamProvider ReactRouterRoute={Route}>
+          <Route
+            path="/:indicator?/:level?/:year?/:tus?/:module?"
```

Comment:
Should this be removed?

Reply:
Yes, leftovers from react-router-dom tests